packages/app/src/systems/Bridge/components/AccountSwitch/AccountSwitch.tsx

```diff
@@ -0,0 +1,81 @@
+import { cssObj } from '@fuel-ui/css';
+import {
+  Button,
+  Dialog,
+  Stack,
+  Text,
+  Flex,
+  Image,
+  Copyable,
+} from '@fuel-ui/react';
+
+import { formatAddress } from '~/systems/Core/utils';
+
+type AccountSwitchProps = {
+  open: boolean;
+  accounts: string[];
+  onSelect: (val: string) => void;
+  onConnect: () => void;
+  onDisconnect: () => void;
+};
+
+export const AccountSwitch = ({
+  open,
+  accounts,
+  onSelect,
+  onConnect,
+  onDisconnect,
+}: AccountSwitchProps) => {
+  const accountsList = accounts.map((account) => {
```

Comment:
no needs to create an `accountList` and then inserting it later. we can put directly inside the other elements